docs/standard/glossary.md

```diff
@@ -86,7 +86,7 @@ See [CLR](#clr)
 
 In contrast to the [CLR](#clr), CoreRT is not a virtual machine, which means it doesn't include the facilities to generate and run code on-the-fly because it doesn't include a [JIT](#jit). It does, however, include the [GC](#gc) and the ability for run-time type identification (RTTI) and reflection. However, its type system is designed so that metadata for reflection isn't required. Not requiring metadata enables having an [AOT](#aot) tool chain that can link away superfluous metadata and (more importantly) identify code that the app doesn't use. CoreRT is in development.
 
-See [Intro to .NET Native and CoreRT](https://github.com/dotnet/corert/blob/master/Documentation/intro-to-corert.md).
+See [Intro to .NET Native and CoreRT](https://github.com/dotnet/corert/blob/main/Documentation/intro-to-corert.md).
```

Comment:
@stephentoub Can you point to the file in dotnet/runtime that should replace this link?